CHANGELOG.rst

```diff
@@ -1,6 +1,28 @@
 Changelog
 =========
 
+
+1.15.1 (2023-03-09)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Allow and prefer non-prefixed extra fields for ``AzureDataFactoryHookAsync``.
+  (`#899 <https://github.com/astronomer/astronomer-providers/pull/899>`_)
+- Pass connection_id in ``HttpTrigger``.
```

Comment:
Fix `HttpSensorAsync` to use the correct connection id instead of a default connection id

Reply:
added it.